### PR TITLE
Minor fixes for quantifiers rewriter

### PR DESCRIPTION
**Pull request by ajreynol**
Fixes related to pools and patterns.

### CODE REVIEW

File: src/theory/quantifiers/quantifiers_rewriter.cpp [hunk 1]
```diff
@@ -175,12 +175,29 @@ void QuantifiersRewriter::computeArgVec2(const std::vector<Node>& args,
...
+  if (!activeMap.empty())
+  {
+    for (const Node& a : args)
+    {
+      if (activeMap.find(a) != activeMap.end())
+      {
+        activeArgs.push_back(a);
       }
     }
   }
```
Comment (HanielB) [inline src/theory/quantifiers/quantifiers_rewriter.cpp:203]:
```suggestion
  if (!varComputePatList)
  {
    for (const Node& ip : ipl)
    {
      Kind k = ip.getKind();
      if (k == INST_ADD_TO_POOL || k == SKOLEM_ADD_TO_POOL)
      {
        varComputePatList = true;
        break;
      }
    }
  }
  else
  {
    for (const Node& a : args)
    {
      if (activeMap.find(a) != activeMap.end())
      {
        activeArgs.push_back(a);
      }
    }
  }
  if (varComputePatList)
  {
    computeArgs( args, activeMap, ipl, visited );
  }
```

**Review by HanielB — APPROVED** [review]
LGTM, just a minor suggestion

File: src/theory/quantifiers/quantifiers_rewriter.cpp (hunk 1 above)
Comment (ajreynol) [reply src/theory/quantifiers/quantifiers_rewriter.cpp:203]: This changes the behavior :)  The issue is that `computeArgs` updates `activeMap`.

Comment (HanielB) [reply src/theory/quantifiers/quantifiers_rewriter.cpp:203]: haha I had stared at it for a while trying to convince myself it'd not. Sorry :)

**Pull request merged**